src/Sulu/Bundle/AdminBundle/Resources/js/components/Grid/README.md

```diff
@@ -0,0 +1,104 @@
+The grid system consists out of 3 components: `Grid`, `Section` and `Item`. The `Grid` component is the overall
+container and should always be used if you want to create a grid. The `Item` subcomponent can mostly be treated like a
+column in an usual grid system. If the sum of the item sizes exceeds 12 the next item will start in a new line.
+A `Section` serves as a container for `Item` components. 
```

Comment:
Maybe also mention what the difference to an `Item` is (I think they have a different margin?)